WebApplication/Migration.cs

```diff
@@ -109,12 +109,17 @@ private async Task ScanBucket(List<MigrationJob> migrationJobs, string bucketKey
                // new project list does not contain old project => lets copy and adopt
                var ossAttributes = new OssAttributes(projectName);
                string metadataFile = ossAttributes.Metadata;
-               ProjectMetadata projectMetadata = await bucketOld.DeserializeAsync<ProjectMetadata>(metadataFile);
```

Comment:
This is strange that project does not have metadata, because metadata presents from very beginning.
Maybe metadata present, but it cannot deserialize `ProjectMetadata`? (say - some field is missing).
Do you have a bucket with old data to check what really happened?

Reply:
I have checked that bucket and the file was really missing. There was only the project zip file, nothing else. Anyway this helps in whatever case (not just in case of missing file), it reports the problem and let the migration continue, so I believe it is a good approach.

Reply:
To clarify: I agree about "bullet-proof" approach, so it will not fail unexpectedly. :)
Was trying to understand why it might happen.

Reply:
I have no idea, probably a crash during adoption.